src/amo/components/Link.js

```diff
@@ -18,7 +19,12 @@ export class LinkBase extends React.Component {
   }
 
   render() {
-    const { base, children, to } = this.props;
+    const { base, children, href, to } = this.props;
+
+    if (typeof href === 'string' && href.startsWith('/')) {
+      const linkHref = path.join(base, href);
+      return <Link {...this.props} href={linkHref}>{children}</Link>;
```

Comment:
This adds too much duplicate code which introduces a maintenance burden and possibly a debugging headache one day.

I would like to see the calling code updated to pass in `to` like `<Link to={`/addon/${addon.slug}/privacy/`} ... />`.

If you really think we should support `href` then I'd like to ideally convert all `<Link>` calls to use `href` (could be done in a follow-up) and/or use more of an aliasing approach to cut down on duplicated code. For example:
````js
let linkTo = to;
if (linkTo === undefined and href !== undefined) {
  linkTo = href;
}
if (typeof linkTo === 'string' && linkTo.startsWith('/')) {
  // ...
}
````